net-dns/libidn2/libidn2-2.3.4.ebuild

```diff
@@ -29,7 +29,6 @@ DEPEND="
 "
 BDEPEND="
 	dev-lang/perl
-	sys-apps/help2man
```

Comment:
It looks, to me, to be an error that we're not installing the man pages for these actually.

Debian does install these: https://packages.debian.org/sid/amd64/libidn2-dev/filelist.

Looks like `--disable-doc` was added in b599dba8ad5aa173fc5459b4e6b87a1de62d7075 but I don't see why yet.

Reply:
Please put a TODO there and maybe file a bug for it, I think we need to fix that properly (I can handle it if you want).

Reply:
sys-libs/pam downloads a docs tarball with pre-generated docs in there but it doesn't seem to install any of it, maybe in the past man pages were in there too, so yeah, that's suspicious.

dbus-glib seems to symlink some doc stuff around and I have /usr/share/gtk-doc/* installed from it, so that's fine

I'll file a bug for libidn2 + pam